app/src/main/java/de/tum/in/tumcampusapp/utils/DateTimeUtils.kt

```diff
@@ -49,18 +51,18 @@ object DateTimeUtils {
             diff < 60 * MINUTE_IN_MILLIS -> {
                 val formatter = DateTimeFormat.forPattern("m")
                         .withLocale(Locale.ENGLISH)
-                "${context.getString(R.string.IN)} ${formatter.print(DateTime(diff))} " +
+                "${context.getString(R.string.IN)} ${formatter.print(DateTime(diff))}" +
                         context.getString(R.string.MINUTES)
             }
-        // Be more precise by telling the user the exact time if below 3 hours
-            diff < 3 * HOUR_IN_MILLIS -> {
-                val formatter = DateTimeFormat.forPattern("HH:mm")
+            diff < 5 * HOUR_IN_MILLIS -> {
+                val formatter = DateTimeFormat.forPattern("H 'hrs' mm 'mins'")
                         .withLocale(Locale.ENGLISH)
```

Comment:
I'd like to edit this function again to round up the left minutes to hours so that it could show the left time more precisely. (https://github.com/TUM-Dev/Campus-Android/issues/1478)